src/test/fuzz/tx_pool.cpp

```diff
@@ -143,7 +143,7 @@ FUZZ_TARGET_INIT(tx_pool_standard, initialize_tx_pool)
     CTxMemPool tx_pool_{/*estimator=*/nullptr, /*check_ratio=*/1};
     MockedTxPool& tx_pool = *static_cast<MockedTxPool*>(&tx_pool_);
 
-    chainstate.SetMempool(&tx_pool);
+    chainstate.SetMempool(tx_pool);
```

Comment:
`m_mempool` can be accessed directly.

```suggestion
    chainstate.m_mempool = &tx_pool;
```

Reply:
thanks, done